### PR TITLE
porting product_lot 10.0 -> 11.0

### PR DESCRIPTION
**Pull request by Enigma228322**
Signed-off-by: Vildan Safin <vildan.safin.99@gmail.com>

### CODE REVIEW

**Comment by Enigma228322** [comment]
@yelizariev @Ramil-Mukhametzyanov pr needs review

File: product_lot/doc/index.rst [hunk 1]
```diff
@@ -8,7 +8,8 @@ Installation
 
 Configuration
 =============
-Activate product variants in Sales app settings.
+* Open ``Inventory >> Settings``
+Activate product ``Attributes and Variants``.
```
Comment (KolushovAlexandr) [inline product_lot/doc/index.rst]: * wrong indent
* activate **product**??

File: product_lot/views.xml [hunk 2]
```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2014-2017 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+     Copyright 2017-2018 Artyom Losev <https://github.com/ArtyomLosev>
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html). -->
+
 <openerp>
```
Comment (KolushovAlexandr) [inline product_lot/views.xml]: `<odoo>`

File: product_lot/models.py [hunk 3]
```diff
@@ -1,3 +1,9 @@
+# Copyright 2014-2017 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+# Copyright 2019 Vildan Safin <https://it-projects.info/team/yelizariev>
```
Comment (KolushovAlexandr) [inline product_lot/models.py]: sorting done wrong. From first to last ones

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]
qty on hand is not updated after `split lot` button clicked

File: product_lot/views.xml [hunk 4]
```diff
@@ -1,19 +1,23 @@
...
-                </xpath>
-            </field>
-        </record>
-</openerp>
+<!-- Copyright 2014-2017 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+     Copyright 2017-2018 Artyom Losev <https://github.com/ArtyomLosev>
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html). -->
+
+<odoo>
+    <record id="product_lot_normal_form_view" model="ir.ui.view">
```
Comment (KolushovAlexandr) [inline product_lot/views.xml]: Keep commit history clean. Do not change what may not be changed. Leave indents for `<record>...` as they were

**Review by KolushovAlexandr — CHANGES_REQUESTED** [review]
* Qty on hand is updated after `splitting lot`.
* Forecasted qty is updated twice
For example, if we have 5 'computer case' in 'ice cream' (on hand and forecasted for both of them are 0) then after splitting the lot we have
*  -1 forecasted for 'ice cream'
* 5 on hand for 'computer case'
* 10 forecasted for 'computer case'

@ilmir-k I'm not sure about the first option Is it how it should be?

@Enigma228322 
forecasted should not be updated twice after `splitting`

**Pull request closed without merging**